.gitignore

```diff
@@ -1 +1,34 @@
+# phpstorm project files
+.idea
+
+# netbeans project files
+nbproject
+
+# zend studio for eclipse project files
+.buildpath
+.project
+.settings
+
+# sublime text project / workspace files
+*.sublime-project
+*.sublime-workspace
+
+# visual studio code project files
+.vscode
+
+# windows thumbnail cache
+Thumbs.db
+
+# Mac DS_Store Files
+.DS_Store
+
+# composer vendor dir
```

Comment:
hm.. this stuff should belong to a global gitignore file: 

https://help.github.com/articles/ignoring-files/#create-a-global-gitignore